pkgs/data/themes/sweet-nova/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  fetchFromGitHub,
+  gitUpdater,
+  lib,
+  stdenvNoCC,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "sweet-nova";
+  version = "unstable-2023-04-02";
+
+  src = fetchFromGitHub {
+    owner = "EliverLara";
+    repo = "Sweet";
+    rev = "8a5d5a7d975567b5ae101b9f9d436fb1db2d9b24";
+    hash = "sha256-FVcXBxcS5oFsvAUDcwit7EIfgIQznl8AYYxqQ797ddU=";
+  };
+
+  sourceRoot = "${src.name}/kde";
```

Comment:
Isn't better just `cd kde`?

Reply:
Sure, if you think it's better. Changed!